### DISCUSSION TITLE
How to define custom url  port for production?

### DISCUSSION THREAD

**Discussion by MeditatorLiu** [serenity-is/Serenity#6998] [Help] [answered]
How to define the port of my published application? The default port of the deployed application is 5000 & 5001, but I don't know where to define the port.
Thanks for your help.

**Comment by volkanceylan** [comment]
https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints?view=aspnetcore-7.0

**Comment by reach2rv** [comment, marked as answer]
> How to define the port of my published application? The default port of the deployed application is 5000 & 5001, but I don't know where to define the port.
> Thanks for your help.

Add "Urls":"http://*:port" to app settings

Make sure you define unique port for each instance
